src/Microsoft.PowerShell.PSReadLine/ConsoleLib.cs

```diff
@@ -386,13 +403,16 @@ public static string ToGestureString(this ConsoleKeyInfo key)
                 sb.Append("Alt");
             }
 
-#if UNIX
             char c = key.KeyChar;
-#else
-            // Windows cannot use KeyChar as some chords (like Ctrl+[) show up as control characters.
-            char c = ConsoleKeyChordConverter.GetCharFromConsoleKey(key.Key,
-                (mods & ConsoleModifiers.Shift) != 0 ? ConsoleModifiers.Shift : 0);
+#if !UNIX
+            if (_toUnicodeApiAvailable)
```

Comment:
If you want, I can add specialized code so that if those apis aren't available, we don't have those bindings by default.  Seems like over optimizing for what is today an edge case.  Alternatively, I could have PSReadline fail to load if those apis aren't available even though most of the PSReadline capability seems to run fine on Nano.  Seems like the best option is to document this and have Nano team make the call to add that api.